cernopendata/modules/theme/assets/semantic-ui/scss/visualise.scss

```diff
@@ -28,6 +28,29 @@
   }
 }
 
+#parameter-button-row .item {
+  margin-left: 0.5em;
+}
+
+#vis-histograms .plot-control {
```

Comment:
Since you're modifying this file, what do you think about arranging it a little bit better? Developers that touch this file in the future will be grateful :)

- We can use SASS nesting capabilities to move things under a single `#vis-histograms`
- Colors can be moved to variables file to avoid _magic numbers_
- It's [not recommended to use `id`s](https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4) unless it's completely necessary, they can be converted to classes. If you do this, remember to change JS selectors as well.

Reply:
Thanks for the tips, I've addressed most of the items, however still left some id`s which were queried in jQuery quite frequently